docker/docker.go

```diff
@@ -271,3 +272,22 @@ func IsSupportedDockerVersion(info types.Info, K8sVersion string) (bool, error)
 	}
 	return false, nil
 }
+
+func ReadFileFromContainer(ctx context.Context, dClient *client.Client, hostname, container, filePath string) (string, error) {
+	content, _, err := dClient.CopyFromContainer(ctx, container, filePath)
+	if err != nil {
+		return "", fmt.Errorf("Failed to copy file [%s] from container [%s] on host [%s]: %v", filePath, container, hostname, err)
+	}
+	tarReader := tar.NewReader(content)
+	if _, err := tarReader.Next(); err != nil {
+		return "", err
+	}
+	file, err := ioutil.ReadAll(tarReader)
+	if err != nil {
+		return "", err
+	}
+	if err := content.Close(); err != nil {
```

Comment:
should we close the content when errors on line 282 and 287 happen? If yes, do with `defer`